oak_runtime/src/introspect.rs

```diff
@@ -120,6 +122,23 @@ fn handle_request(
             "Object Counts",
             &runtime.html_counts(),
         ))));
+    // Endpoint to load the list introspection events, serialized via protobuf
+    } else if path == "/introspection-events" {
+        let events_message = Events {
+            events: Vec::from(runtime.introspection_event_queue.lock().unwrap().clone()),
+        };
+
+        let mut buffer: Vec<u8> = Vec::new();
+        events_message.encode(&mut buffer).unwrap();
+
+        let mut response = Response::new(Body::from(buffer));
+        // Set the Access-Control-Allow-Origin header, so this endpoint can be
+        // called from within a browser.
+        response
+            .headers_mut()
+            .insert(ACCESS_CONTROL_ALLOW_ORIGIN, HeaderValue::from_static("*"));
```

Comment:
Allows all origins to call this, the least secure policy, but seems fair for our case.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Origin